sherpa-onnx/csrc/session.cc

```diff
@@ -214,4 +220,13 @@ Ort::SessionOptions GetSessionOptions(
   return GetSessionOptionsImpl(config.num_threads, config.provider);
 }
 
+void OrtStatusFailure(OrtStatus *status,std::ostringstream *os) {
```

Comment:
This function is used only inside this file, so you can remove it from the header file, and move this function to the beginning of this file and mark it as `static`.

Reply:
ok I will make the changes